.github/workflows/PaperBuild.yml

```
@@ -0,0 +1,39 @@
+name: Build paper
+
+on:
+  push
```

```suggestion
on: [push, pull_request]
```
This will probably solve your problem.